awx/main/analytics/core.py

```diff
@@ -25,6 +26,8 @@
 
 def _valid_license():
     try:
+        if get_license(show_key=False)['license_type'] == 'open':
```

Comment:
If there's no license, `license_type` won't be a valid key.  Use:

`.get('license_type', 'UNLICENSED')`